### PR TITLE
upgrade grpc-tools and dotnet version

### PR DESCRIPTION
**Pull request by liuhaoyang**
Supports .NET 5 and .NET 6, and no longer supports .NET Framework and .NET Core 2.0

### CODE REVIEW

**Comment by wu-sheng** [comment]
Please update the document, such as https://github.com/SkyAPM/SkyAPM-dotnet#supported

**Comment by liuhaoyang** [comment]
> Please update the document, such as https://github.com/SkyAPM/SkyAPM-dotnet#supported

updated.

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ SkyAPM C#/.NET instrument agent
 > MyGet feed URL https://www.myget.org/F/skyapm-dotnet/api/v3/index.json
 
 # Supported
-- This project currently supports apps targeting netcoreapp2.0/netframework4.6.1 or higher.
+- This project currently supports apps targeting netcoreapp3.1、net5.0、net6.0 or higher.
```
Comment (wu-sheng) [inline README.md:31]: As this platform level upgrade, should we update the dotnet agent to 2.x for the next release?
This seems a big break change.

Comment (liuhaoyang) [reply README.md:31]: I agree. I will also support the skywalking log protocol in the agent, and then release version 2.0.

**Comment by wu-sheng** [comment]
Then, I am renaming the milestone to 2.0

**Review by wu-sheng — APPROVED** [review]

**Comment by wu-sheng** [comment]
Please update the snapshot version number through another pull request.

**Pull request merged**